src/db/QDjangoWhere.h

```diff
@@ -61,6 +61,13 @@ class QDJANGO_DB_EXPORT QDjangoWhere
         IContains
     };
 
+    enum AggregateType{
+        AVG,
```

Comment:
I am undecided as to the right name for these. The uppercase name matches SQL nicely, but doesn't really look very "Qt-ish".